### PR TITLE
Point match client

### PR DESCRIPTION
**Pull request by fcollman**
Here's a point match client to address #68

### CODE REVIEW

File: renderapi/client.py [hunk 1]
```diff
@@ -859,3 +860,192 @@ def transformSectionClient(stack, transformId, transformClass, transformData,
...
+        of transformations, and you had labelled the first non-local transform 'montage' then setting
+        excludeFirstTransformAndAllAfter='montage' would remove that montage transform and any other
+        transforms that you had applied after it. default= None.
+    excludeAllTransforms: bool
+        alternative to normalizeForMatching which simply removes all transforms from the list.
+        default=False
+
+    """
+    if pointMatchRender is None:
+        pointMatchRender = Render(host, port, owner, project, client_script)
```
Comment (RussTorres) [inline renderapi/client.py]: This, I think, does not allow us to overwrite with specified host, port, owner parameters if pointMatchRender is not None.  because this is not related to a regular render option it is not a huge departure from the way the rest of the module works, but still a little counterintuitive.

Comment (fcollman) [reply renderapi/client.py]: hmm.. good point... but this is the only cross tilespec, point match api call in one call.... so some possibilities.  1) don't allow overwriting at all here.. have tilespecRender and pointmatchRender (optionally) 2) have host and pm_host, port and pm_port, etc...  3) ??

File: renderapi/client.py [hunk 2]
```diff
@@ -859,3 +859,251 @@ def transformSectionClient(stack, transformId, transformClass, transformData,
...
+
+    def to_java_args(self):
+        args = []
+        for key, value in self.__dict__.items():
+            if (value is not None) and not (key == 'kwargs'):
+                args += self.get_cmd_opt(value, "--{}".format(key))
+        return self.sanitize_cmd(args)
+
+
+class FeatureRenderParameters(ArgumentParameters):
```
Comment (fcollman) [inline renderapi/client.py]: this is not being used.

Comment (RussTorres) [reply renderapi/client.py]: I removed those parameters -- these will be easy enough to add in if we want to use this pattern for additional or existing clients
